### DISCUSSION TITLE
npm run generate with static filenames in _nuxt directory?

### DISCUSSION THREAD

**Discussion by gritworks** [nuxt/nuxt#24589] [Questions] [answered]
### Describe the feature

Hello,

Nuxt3 config: Tried to get the filenames statically generated within the _nuxt folder (to be loaded by a .php file) when running "npm run generate" but nothing worked.
Vite is the bundler but has only a build property.

I could not find a suitable solution for my problem in the docs and googling also didn't provide any working solution.

Am i missing something?

Kind regards, Wouter

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by danielroe** [comment, marked as answer]
You would do something like this:

```ts
export default defineNuxtConfig({
  vite: {
    build: {
      rollupOptions: {
        output: {
          chunkFileNames: '_nuxt/chunks/[name].js',
          entryFileNames: '_nuxt/js/[name].js',
          assetFileNames: '_nuxt/[name].[ext]'
        }
      }
    }
  }
})
```
